src/test/java/jenkins/plugins/git/GitSampleRepoRule.java

```diff
@@ -63,7 +63,7 @@ private static void checkGlobalConfig() throws Exception {
     public void init() throws Exception {
         run(true, tmp.getRoot(), "git", "version");
         checkGlobalConfig();
-        git("init");
+        git("init", "--template="); // initialize without copying the installation defaults to ensure a vanilla repo that behaves the same everywhere
```

Comment:
was tested on windows - this skips using a template without warning.

ran `git init` in an empty directory and checked the console output and the files in `.git/`  (observe there are default system hooks copied if you have any configure such as `git-secrets`)

ran `git init --template=` in an empty directory and checked the console output and the files in `.git/`  (observe there is no `hooks` directory)